tests/kafkatest/services/kafka/kafka.py

```diff
@@ -575,10 +577,21 @@ def set_unclean_leader_election(self, topic, value=True, node=None):
         node.account.ssh(cmd)
 
     def _connect_setting_kafka_configs(self, node):
+        # Use this for everything related to kafka-configs except User SCRAM Credentials
         if node.version.kafka_configs_command_uses_bootstrap_server():
-            return "--bootstrap-server %s " % self.bootstrap_servers(self.security_protocol)
+            return "--bootstrap-server %s --command-config <(echo '%s')" % (self.bootstrap_servers(self.security_protocol),
```

Comment:
It's sort of weird that we're creating files on the fly with '<(echo '%s')` rather than just using a file in /mnt/security.  This will result in some pretty long command lines, right?

Considering this is an existing pattern in kafka.py (but nowhere else?), let's file a follow-on JIRA to look into this and fix it.  Unless there's some really good reason why `kafka.py` is doing this, but I can't think of any.

Reply:
I created https://issues.apache.org/jira/browse/KAFKA-10451 to track this.